onto-viewer-core/src/test/java/org/edmcouncil/spec/ontoviewer/core/ontology/data/label/provider/DefaultSettingsTest.java

```diff
@@ -0,0 +1,100 @@
+package org.edmcouncil.spec.ontoviewer.core.ontology.data.label.provider;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.xpath.XPathExpressionException;
+import org.semanticweb.owlapi.apibinding.OWLManager;
+import org.edmcouncil.spec.ontoviewer.configloader.configuration.model.impl.ViewerCoreConfiguration;
+import org.edmcouncil.spec.ontoviewer.core.ontology.OntologyManager;
+import org.junit.jupiter.api.BeforeEach;
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.semanticweb.owlapi.model.IRI;
+import org.semanticweb.owlapi.model.OWLOntologyCreationException;
+import org.semanticweb.owlapi.model.OWLOntologyManager;
+import org.xml.sax.SAXException;
+import org.semanticweb.owlapi.model.OWLOntology;
+
+/**
+ *
+ * @author patrycja.miazek (patrycja.miazek@makolab.com)
+ */
+public class DefaultSettingsTest {
+
+  @TempDir
+  Path tempDir;
+
+  private LabelProvider labelProviderTest;
+
+  @BeforeEach
+  public void setUp() throws URISyntaxException, IOException, OWLOntologyCreationException, ParserConfigurationException, XPathExpressionException, SAXException {
+
```

Comment:
In this method and other `@BeforeEach` methods in other tests, there is a lot of common code. I would suggest moving it to a new abstract superclass from which all these test classes will inherit.